packages/google_maps_flutter/google_maps_flutter_android/lib/src/google_maps_flutter_android.dart

```diff
@@ -625,6 +631,22 @@ class GoogleMapsFlutterAndroid extends GoogleMapsFlutterPlatform {
     GoogleMapsInspectorPlatform.instance =
         GoogleMapsInspectorAndroid((int mapId) => _channel(mapId));
   }
+
+  /// Intializes Google Maps SDK for Android with preferred renderer
+  ///
```

Comment:
This comment needs to explain that this will only work if it's called before the creation of any map instances.